item-spec/common-metadata.md

```diff
@@ -221,15 +221,15 @@ with the links section, to describe things like the Catalog an Item is in, relat
-| rel        | string | **REQUIRED.** Relationship between the current document and the linked document. See chapter "Relation types" for more information.                                            |
-| type       | string | [Media type](../catalog-spec/catalog-spec.md#media-types) of the referenced entity.                                                                                            |
-| title      | string | A human readable title to be used in rendered displays of the link.                                                                                                            |
-| method     | string | The HTTP method required to be supported by the target resource. `GET` by default                                                                                              |
-| headers    | object | The HTTP headers required to be sent by the target resource.                                                                                                                   |
-| body       | object | The HTTP body required to be sent by the target resource.                                                                                                                      |
+| Field Name | Type           | Description                                                                                                                                                                    |
+| ---------- | -------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| href       | string         | **REQUIRED.** The actual link in the format of an URL. Relative and absolute links are both allowed. [Trailing slashes are significant.](../best-practices.md#consistent-uris) |
+| rel        | string         | **REQUIRED.** Relationship between the current document and the linked document. See chapter "Relation types" for more information.                                            |
+| type       | string         | [Media type](../catalog-spec/catalog-spec.md#media-types) of the referenced entity.                                                                                            |
+| title      | string         | A human readable title to be used in rendered displays of the link.                                                                                                            |
+| method     | string         | The HTTP method required to be supported by the target resource. `GET` by default                                                                                              |
+| headers    | Map<string, *> | The HTTP headers required to be sent by the target resource.                                                                                                                   |
```

Comment:
I think we still need to explain what we discussed in the last call. Arrays will be expanded to multiple headers.

Ultimately, we could also say:
`Map<string, string | [string]>`

Reply:
explaination added

Reply:
Can't find the explanation. Where can it be found?